src/auth/src/credentials/internal/sts_exchange.rs

```diff
@@ -257,7 +248,7 @@ mod test {
                 issued_token_type: ACCESS_TOKEN_TYPE.to_string(),
                 token_type: "Bearer".to_string(),
                 expires_in: 3600,
-                scope: "https://www.googleapis.com/auth/cloud-platform".to_string(),
+                scope: Some("https://www.googleapis.com/auth/cloud-platform".to_string()),
```

Comment:
Should we use the constant here?

Reply:
oh, just found that is declared in `src/auth/src/credentials/service_account.rs`

Reply:
Maybe we can define this in a common place. Not sure what is the usual way to do in Rust.

How about creating a `Constants.rs` mod under `src/auth`?

Reply:
will merge this and send a PR creating the constants file to not mix with the main goal of this PR